src/coins.cpp

```diff
@@ -517,30 +518,39 @@ void CCoinsViewCache::PopHistoryNode(uint32_t epochId) {
             // back.
             
             // Sensible action is to truncate the history cache:
+        }
```

Comment:
The braces are a bit misleading, since we actually depends on fall-through here. I used to have a convention of writing `//fallthru` in that case. In C++17 (which we can't use quite yet) it is written `[[fallthrough]];`, which can be used to suppress a compiler warning.

Reply:
I understand why this was done now; it's necessary in order to declare the local `tmpHistoryRoot`.